internal/dashboard/handler/obcluster_handler.go

```diff
@@ -275,3 +275,26 @@ func DeleteOBZone(c *gin.Context) (any, error) {
 	}
 	return nil, nil
 }
+
+// @ID ListEssentialParameters
+// @Summary list essential parameters
+// @Description list essential parameters of specific obcluster
+// @Tags Obcluster
+// @Accept application/json
+// @Produce application/json
+// @Param namespace path string true "obcluster namespace"
+// @Param name path string true "obcluster name"
+// @Success 200 object response.APIResponse{data=response.OBClusterEssentialParameters}
+// @Failure 400 object response.APIResponse
+// @Failure 401 object response.APIResponse
+// @Failure 500 object response.APIResponse
+// @Router /api/v1/obclusters/{namespace}/{name}/essential-parameters [GET]
+// @Security ApiKeyAuth
+func ListEssentialParameters(c *gin.Context) (*response.OBClusterEssentialParameters, error) {
```

Comment:
Parameters usually means configs for observer or tenant, and it can be set by alter system